sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -565,6 +576,8 @@ class annotated_ref {
     annotated_ref(T*);
     annotated_ref(const annotated_ref&) = default;
     operator T() const;
+    operator T&();
```

Comment:
We should not return a non-annotated reference. This break the whole reason why we have annotated-ref (to make sure annotations don't get silently dropped).

Reply:
removed